### PR TITLE
🔍  Informations, Robots, SEO improvement and background pattern changed.

### PR DESCRIPTION
**Pull request by GMBermeo**

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Allow: /
```
Comment (GMBermeo) [inline public/robots.txt:2]: Not sure about this syntax.

File: README.md [hunk 2]
```diff
@@ -62,16 +62,16 @@ yarn build
...
 - ![#AEFAFF](https://via.placeholder.com/15/AEFAFF/AEFAFF.png) `#AEFAFF`
 - ![#FF7A00](https://via.placeholder.com/15/FF7A00/FF7A00.png) `#FF7A00`
   - ![#FF0000](https://via.placeholder.com/15/FF0000/FF0000.png) `#FF0000`
 
 ## Links
 
-- <https://a1z26.bermeo.dev>
-  - <https://soloculto.bermeo.dev>
-  - <https://a1z26.vercel.app>
+- [A1Z26.Bermeo.dev](https://a1z26.bermeo.dev)
```
Comment (GMBermeo) [inline README.md:72]: Better formating for lint.

Comment (andersonsf90) [reply README.md:72]: xo ti muda aquele negócio lá

**Review by andersonsf90 — APPROVED** [review]
Good job!

**Review by GMBermeo — COMMENTED** [review]
Changed width file.

**Comment by andersonsf90** [comment]
é vapo!

**Review by andersonsf90 — APPROVED** [review]

**Pull request merged**